package.json

```diff
@@ -44,28 +40,30 @@
   "license": "MIT",
   "dependencies": {
     "css-mediaquery": "^0.1.2",
-    "expo-screen-orientation": "~1.1.1"
+    "expo-screen-orientation": "~2.1.0"
   },
   "peerDependencies": {
-    "@unimodules/core": "~5.3.0",
-    "react-native": "~0.62"
+    "@unimodules/core": "*",
+    "react-native": "*"
   },
```

Comment:
let's just remove peer deps entirely - this can be problematic with npm 7. related: https://github.com/expo/expo/pulls?q=is%3Apr+is%3Aclosed